libs/ledger-live-common/src/families/hedera/tests/js-estimateMaxSpendable.test.ts

```diff
@@ -50,8 +51,27 @@ const account: Account = {
   },
 };
 
-describe("js-estimateMaxSpendable", () => {
-  const estimatedFees = new BigNumber("83300").multipliedBy(2);
+describe("js-estimateMaxSpendable", async () => {
+  // If get hedera price works, use real estimate, otherwise fallback to hard coded
+  let hederaPrice;
+
+  try {
+    const { data } = await axios.get(
```

Comment:
Because you add some network logic we will need to change it to an integration test.
It's our way to not have network dependency for our CI.
can you rename js-estimateMaxSpendable.test.ts to js-estimateMaxSpendable.integration.test.ts ?

Or you can also change it with a mock test with predicted data

Reply:
This makes sense, will change soon.  Thanks.  I believe that it is desired that we check using the network dependency as this test is currently failing because the value has changed.  To prevent future failures we will need to check a dynamic value to determine the fees.

Reply:
Sorry for the delay, I had accidentally changed this on the wrong branch at first and didn't notice, but it is changed now